### ISSUE TITLE
Any city output (science/culture/etc.) will drop to zero, if it exceeds a certain value.

### ISSUE THREAD

**Issue by Qwest8K** [caveman2cosmos/Caveman2Cosmos#1604]
This value is somewhere between 210,000 and 220,000:

![85](https://github.com/user-attachments/assets/e476569b-2def-4484-b144-5382f8f39c85) ![90](https://github.com/user-attachments/assets/95038c66-ef9f-49f8-8de0-288590601579)

**Comment by flabbert** [comment]
I think we need a savegame for these, can you provide one? It will probably yield some interesting information with regards to how game functions around this

**Comment by flabbert** [comment]
teagging #1605 as related

**Comment by Qwest8K** [comment]
> I think we need a savegame for these, can you provide one? It will probably yield some interesting information with regards to how game functions around this

Yes, for both issues: [Saves for 1604-1605.zip](https://github.com/user-attachments/files/17784365/Saves.for.1604-1605.zip)

**Comment by flabbert** [comment]
I also assume you use latest svn?

**Comment by Qwest8K** [comment]
> I also assume you use latest svn?

Yes.
